packages/common/package.json

```diff
@@ -12,6 +12,7 @@
   "author": "Temporal Technologies Inc. <sdk@temporal.io>",
   "license": "MIT",
   "dependencies": {
+    "@grpc/grpc-js": "^1.12.4",
```

Comment:
Let's not add a dependency to gRPC here if we can avoid.

Reply:
Thanks, done.